README.rst

```diff
@@ -16,18 +16,10 @@ MDAnalysis also includes widely used analysis algorithms in the `MDAnalysis.anal
 The MDAnalysis project uses an `open governance model`_ and is fiscally sponsored by `NumFOCUS`_. Consider making 
 a `tax-deductible donation`_ to help the project pay for developer time, professional services, travel, workshops, and a variety of other needs.
 
-.. raw:: html
-
-   <div align="center">
-      <a href="https://numfocus.org/project/mdanalysis">
-         <img height="60px" 
-            src="https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png"
-            alt="NumFOCUS Logo"
-            align="center">
-      </a>
-   </div>
-   <br />
-
+.. image:: https://www.mdanalysis.org/public/images/numfocus-sponsored-small.png
+   :alt: NumFOCUS (Fiscally Sponsored Project)
+   :target: https://numfocus.org/project/mdanalysis
+   :align: center
```

Comment:
For some reason the logo is not center aligned?

Reply:
I recall having some problems centering the logo... https://github.com/MDAnalysis/mdanalysis/pull/2684#discussion_r427188689

Reply:
I don't think that centering works except with the raw html. But the raw html is not allowed on PyPi. I opted for getting rid of almost duplicated files instead of perfect logo placement in the readme.

Reply:
Consider the `:align: center` aspirational... (it is supposed to work if you believe the reST docs but it seems that the PyPi and GitHub renderer just ignore it.)